python_on_whales/components/container/cli_wrapper.py

```diff
@@ -1020,15 +1020,14 @@ def pause(self, containers: Union[ValidContainer, List[ValidContainer]]):
 
         run(full_cmd)
 
-    def prune(self, filters: Union[str, List[str]] = []) -> None:
+    def prune(self, filters: Dict[str, str] = {}) -> None:
         """Remove containers that are not running.
 
         # Arguments
             filters: Filters as strings or list of strings
         """
         full_cmd = self.docker_cmd + ["container", "prune", "--force"]
-        for filter_ in to_list(filters):
-            full_cmd += ["--filter", filter_]
+        full_cmd.add_args_list("--filter", format_dict_for_cli(filters))
```

Comment:
Could you add a unit test for the multiple filters in `docker.container.prune()`?

Reply:
I will put this all in one test function because that seems a bit cleaner. Let me know what you think.